R/normdm.R

```diff
@@ -0,0 +1,51 @@
+#' normdm
+#'
```

Comment:
add a narrative description of what the function accomplishes.  For example:

Simulates groups of normal-distributed data.